test/unit/models_test.rb

```diff
@@ -14,7 +14,7 @@ class ModelsTest < ActiveSupport::TestCase
       'Proxy' => %w[endpoint sandbox_endpoint], 'OIDCConfiguration' => %w[oidc_configurable_type], 'Invitation' => %w[token],
       'Settings' => Switches::SWITCHES.map { |switch| "#{switch}_switch" },
       'Invoice' => %w[pdf_file_name pdf_content_type state friendly_id fiscal_code vat_code currency creation_type], 'DeletedObject' => %w[owner_type object_type],
-      'Onboarding' => %w[wizard_state bubble_api_state bubble_metric_state bubble_deployment_state bubble_mapping_state bubble_limit_state],
+      'Onboarding' => %w[wizard_state],
```

Comment:
@Martouta [this test](https://github.com/3scale/porta/blob/master/test/unit/models_test.rb) is failing because of thie change.
Is it something we have to remove _after_ the migrations?

Reply:
It is normal. You should not change this test. What this is saying is "ignore these attributes, we are intentionally not validating the size of their strings in our code because they are not used"